lib/robotremote.js

```diff
@@ -101,6 +102,11 @@ Server.prototype.getKeywordTags = function (name, response) {
     response(null, this.keywords[name].tags || []);
 };
 
+Server.prototype.getKeywordTypes = function (name, response) {
```

Comment:
other than indirectly, by using libdoc, this could be tested with mocha for basic functionality and maybe couple edge cases.